lib/run_loop/core_simulator.rb

```diff
@@ -558,6 +558,43 @@ def uninstall_app_and_sandbox
     true
   end
 
+  # @!visibility private
```

Comment:
Move this back to private I think.

Reply:
I moved this to none private because I needed this method in client.rb